app/javascript/app/components/country/country-ghg-emissions/country-ghg-emissions-selectors.js

```diff
@@ -345,12 +345,14 @@ export const getChartConfig = createSelector(
     colorThemeCache = { ...theme, ...colorThemeCache };
     const tooltip = getTooltipConfig(yColumnsChecked);
     let unit = DEFAULT_AXES_CONFIG.yLeft.unit;
+    let precision = null;
     if (calculationSelected.value === CALCULATION_OPTIONS.PER_GDP.value) {
       unit = `${unit}/ million $ GDP`;
     } else if (
       calculationSelected.value === CALCULATION_OPTIONS.PER_CAPITA.value
     ) {
       unit = `${unit} per capita`;
+      precision = 2;
```

Comment:
I'm not sure if this is what they want. I will expect the axis to be rounded as it was and the tooltip to show decimals. But right now the decimals are always 0 so it won't help either